sweetest/api/v1/build.gradle

```diff
@@ -10,17 +10,15 @@ plugins {
     id 'kotlin'
 }
 
-apply from: '../publishing.gradle'
-
-setCustomArtifactName('sweetest')
+apply from: '../../publishing.gradle'
+setCustomArtifactName('sweetest-compat-v1')
 
 dependencies {
-    api project(':sweetest:sweetest-base')
-    implementation project(':sweetest:sweetest-core')
+    api project(':sweetest:base')
```

Comment:
an example:

`BaseSteps` derives from `Steps` (in the `base` module) because `core` needs to manage `Steps`. If `BaseSteps` is not exposed as API then `Steps` can't be found during compilation:

```
Cannot access 'com.mysugr.sweetest.internal.Steps' which is a supertype of 'SomeStepsClass'. Check your module classpath for missing or conflicting dependencies
```